src/content/docs/extensions/full-text-search.md

```diff
@@ -77,8 +77,7 @@ CALL CREATE_FTS_INDEX(
     'Book',   // Table name
     'book_index',   // Index name
     ['abstract', 'title'],   // Properties to build FTS index on
-    stemmer := 'porter',   // Stemmer to use (optional)
-    stopwords := 'https://stopwords/porter.txt' // Configure customized stopwords list
+    stemmer := 'porter'   // Stemmer to use (optional)
```

Comment:
We can also mention in the comment that users can pass in stemmer file path too `stemmer := '/stopwords.txt'` ?

Reply:
So we can't pass in a static local `.txt` file? Only https allowed?